### PR TITLE
DCG-2297 align dependencies

### PR DESCRIPTION
**Pull request by bkowalik**

### CODE REVIEW

**Review by teroxik — APPROVED** [review]
This looks good, we have to caerfully roll it out, with the update of the scala version and some of the plugins, so everything still wokrs

File: build.sbt [hunk 1]
```diff
@@ -26,32 +26,30 @@ resolvers ++= Seq(
   Resolver.typesafeRepo("releases"),
   Resolver.typesafeRepo("snapshots"),
   Resolver.sonatypeRepo("snapshots"),
-  "spray repo" at "http://repo.spray.io",
-  "jgit-repo" at "http://download.eclipse.org/jgit/maven"
+  ("spray repo" at "http://repo.spray.io").withAllowInsecureProtocol(true),
```
Comment (teroxik) [inline build.sbt]: do will still need these repos?

File: build.sbt [hunk 2]
```diff
@@ -26,32 +26,30 @@ resolvers ++= Seq(
...
 
-addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.2.1")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0")
 
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.9.0")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.5.2")
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.7.6")
```
Comment (teroxik) [inline build.sbt:45]: any changes caused here or with the sbt assembly plugin?

File: build.sbt [hunk 3]
```diff
@@ -26,32 +26,30 @@ resolvers ++= Seq(
...
 
 addSbtPlugin("com.tapad" % "sbt-docker-compose" % "1.0.35")
```
Comment (teroxik) [inline build.sbt:47]: this one is currently failing us in the CI - if it doesn't go through correctly we don't detect failure

File: build.sbt (hunk 1 above)
Comment (bkowalik) [reply build.sbt]: Removed

**Review by teroxik — APPROVED** [review]

**Pull request merged**